fluent-bit-cloudwatch.go

```diff
@@ -83,6 +83,8 @@ func getConfiguration(ctx unsafe.Pointer, pluginID int) cloudwatch.OutputPluginC
 	logrus.Infof("[cloudwatch %d] plugin parameter auto_create_group = '%v'\n", pluginID, config.AutoCreateGroup)
 	config.CWEndpoint = output.FLBPluginConfigKey(ctx, "endpoint")
 	logrus.Infof("[cloudwatch %d] plugin parameter endpoint = '%s'\n", pluginID, config.CWEndpoint)
+	config.CredsEndpoint = output.FLBPluginConfigKey(ctx, "credentials_endpoint")
+	logrus.Info("[cloudwatch %d] plugin parameter credentials_endpoint = %s\n", pluginID, config.CredsEndpoint)
```

Comment:
This should be `Infof`

Reply:
Thanks!
Will make both the changes and create a new revision